tools/dotprep/dotPrep.xml

```diff
@@ -0,0 +1,84 @@
+<tool id="dotprep" name="dotPrep" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@" license="MIT">
+    <description>Apply unique anchor filtering and prepare Dot coordinates</description>
+        <macros>
+        <token name="@TOOL_VERSION@">1.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+        <token name="@PROFILE@">25.0</token>
+    </macros>
+    <command detect_errors="exit_code"><![CDATA[
```

Comment:
can you please add https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-required-files-include

And remember that we should do that also in other tools over time.
Also add creators, you de.NBI, Galaxy-Freiburg ...